### ISSUE TITLE
Make a migration guide/notes for Jean after migrating stCelo to react-celo to rainbowkit

### ISSUE THREAD

**Issue by nicolasbrugneaux** [celo-org/staked-celo-web-app#126]
Blocked by #125

**Comment by nicolasbrugneaux** [comment]
See draft here: https://github.com/celo-org/celo-monorepo/blob/8e2e2c5c53bd7f7c5df4f2a64974555ad0250615/packages/sdk/contractkit/MIGRATION-TO-VIEM.md

**Comment by nicolasbrugneaux** [comment]
Duplicate of #125 oops
